### DISCUSSION TITLE
Error 404: Page Not Found when opening projects

### DISCUSSION THREAD

**Discussion by lecb** [broadinstitute/seqr#2055] [Q&A]
**Describe the bug**
I have a local installation of seqr. Since yesterday, I get an `Error 404: Page Not Found` error when I open any of the seqr projects. As far as I can tell, nothing has changed.

I have checked everything is looking healthy in docker:

![image](https://user-images.githubusercontent.com/23638585/130103924-876d4df5-45d1-47e6-9dc4-d17123252858.png)

I then killed all docker containers and restarted an instance of seqr `docker-compose up -d seqr`

Everything still says it is healthy....

![image](https://user-images.githubusercontent.com/23638585/130106799-8529c7f4-fcfe-4506-b7b8-109a93db2672.png)

Any help would be appreciated... 
BW
Ellie

**Comment by hanars** [comment]
Can you attach the server logs here?

**Reply by lecb** [reply]
[I presume this is what you need - these are the docker logs for seqr. Sorry, the guy who was helping me with this has now left.... so you may have to be slightly patient with my ignorance! Please let me know if you need something else... 
[server_logs.txt](https://github.com/broadinstitute/seqr/files/7017195/server_logs.txt)

**Reply by lecb** [reply]
Also attaching the postgres logs in case that helps... 
[post_gres_logs.txt](https://github.com/broadinstitute/seqr/files/7017371/post_gres_logs.txt)

**Reply by hanars** [reply]
this looks like a data migration issue, that your database and seqr's database spec are out of date. I would try running
```
docker-compose exec seqr python -u manage.py makemigrations
docker-compose exec seqr python -u manage.py migrate
```

**Reply by lecb** [reply]
You are amazing! It worked! Thanks so much :-D!
